src/filing/common/Footer.css

```diff
@@ -35,6 +35,11 @@
   margin-bottom: 1em;
 }
 
+
+.Filing.App footer.maintenance {
+  padding-bottom: 7rem;
```

Comment:
Can we change this to 6.5rem? There's a little sliver of white between the footer and the maintenance message when fully scrolled down.

Reply:
Updated.